Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:8.1.4
+
+MAINTAINER "HMCTS Reform"
+LABEL maintainer="HMCTS Reform"
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY package.json /usr/src/app/
+RUN npm install
+
+COPY . /usr/src/app/
```

Comment:
don't need to copy a single file earlier. can move this line above the `npm install` command and it should be all good :+1:

Reply:
the reason this is usually done is to speed up cached docker images (i forgot to add the lock file as well)